docs/cases/indicators-of-compromise.asciidoc

```diff
@@ -76,22 +76,31 @@ image::images/indicator-details-flyout.png[Shows the Indicator details flyout, 6
 [[find-related-sec-events]]
 == Find related security events
 
-Investigating an indicator in <<timelines-ui,Timeline>> helps you find related security events in your environment. You can add an indicator to Timeline from the Indicators table or the Indicator details flyout.
+Investigate an indicator in <<timelines-ui,Timeline>> to find related security events in your environment. You can add an indicator to Timeline from the Indicators table or the Indicator details flyout.
 
 [role="screenshot"]
-image::images/indicator-in-timeline.png[Shows the results of an indicator being investigated in Timeline]
+image::images/indicator-query-timeline.png[Shows the results of an indicator being investigated in Timeline]
+
+When you add an indicator to Timeline, a new Timeline opens with an auto-generated KQL query. The query contains the indicator field-value pair that you selected plus the field-value pair of the automactially mapped source event. By default, the query's time range is set to seven days before and seven after the indicator's `timestamp`. 
 
-When you add an indicator to Timeline, a new Timeline opens with a pre-populated KQL query. The query contains the indicator field-value pair that you selected plus the field-value pair of the mapped source event.
+You can use the Timeline query results to identify and predict related events. They can also help you assess and respond to potential threats.
```

Comment:
Maybe move this to the first paragraph, above the image? Might be nice to get a sense of why the Timeline query results would be helpful/useful, before going into nuts & bolts of how it functions.